recipes/pixcdust/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "pixcdust" %}
+{% set version = "0.1.0" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pixcdust-{{ version }}.tar.gz
+  sha256: 75199f9811a198e46c2cb2590e857ea225da8f4ee8430a7568c58899934636b5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >={{ python_min }}
```

Comment:
Please take a look at the `noarch: python` docs here:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

and fix the `python` line here and then also add appropriate entry under `test`/`requires`.